src/server/utilities/cspHeader/index.js

```diff
@@ -47,10 +49,12 @@ const directives = {
       'https://pagead2.googlesyndication.com', // ads
       'https://tpc.googlesyndication.com', // ads
       'https://cdn.ampproject.org',
```

Comment:
Just out of curiosity do we know what the difference is between ampproject.org and ampproject.net?

Reply:
I couldn't determine any differences - they're both owned by Google and redirect to amp.dev.